lib/rspec/core/configuration.rb

```diff
@@ -1613,6 +1613,9 @@ def extract_location(path)
           captures = match.captures
           path, lines = captures[0], captures[1][1..-1].split(":").map { |n| n.to_i }
           filter_manager.add_location path, lines
+        else
+          path, scoped_ids = path.split(/[\[\]]/)
```

Comment:
This group is hard to read with all the slashes and square brackets, do you think it'd be easier to understand if it was `path.split(/[#{Regexp.escape("[]")}]/)`?